### ISSUE TITLE
Directions should be clarified

### ISSUE THREAD

**Issue by beaucarnes** [freeCodeCamp/freeCodeCamp#13994]
The directions on this challenge should be a little more clear. In the description there is the following example of enqueue: 
`PriorityQueue.enqueue(['kitten', 1]);`

There should also be an example of what the result of dequeue should be. I assumed the result of dequeue should be an array since that is what is used for enqueue. After being unable to pass the challenge, I looked at the code that is used to test the challenge and realized that the result of dequeue should be just the item without priority and not the full array passed in to enqueue. 

Maybe it's just me and this would be more obvious to other people.

Challenge [create-a-priority-queue-class](http://beta.freecodecamp.com/en/challenges/coding-interview-data-structure-questions/create-a-priority-queue-class) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```js

function PriorityQueue () {
    var collection = [];
    this.printCollection = function() {
      (console.log(collection));
    };
    this.enqueue = function(element){
        if (this.isEmpty()){ 
            collection.push(element);
        } else {
            var added = false;
            for (var i=0; i<collection.length; i++){
                 if (element[1] < collection[i][1]){ //checking priorities
                    collection.splice(i,0,element);
                    added = true;
                    break;
                }
            }
            if (!added){
                collection.push(element);
            }
        }
    };
    this.dequeue = function() {
        var value = collection.shift();
        return value[0];
    };
    this.front = function() {
        return collection[0];
    };
    this.size = function() {
        return collection.length; 
    };
    this.isEmpty = function() {
        return (collection.length === 0); 
    };
}

```

**Comment by QuincyLarson** [comment]
@beaucarnes you're on to something here. If it is ambiguous to you, then I agree that we should include a couple extra lines clarifying and an example as you suggested. Would you be interested in creating a PR for this?

**Comment by beaucarnes** [comment]
Yeah, I'll do this.

**Comment by beaucarnes** [comment]
It took me a some time to figure this out but apparently the version that is on beta.freecodecamp.com is not the most current version. In the version on Github, this issue is already fixed. Now I know to look at the Github version before creating an issue. :)

**Comment by ko5320** [comment]
2017/3/24 00:50，"Beau Carnes" <notifications@github.com>寫道：

Closed #13994 <https://github.com/freeCodeCamp/freeCodeCamp/issues/13994>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/freeCodeCamp/freeCodeCamp/issues/13994#event-1012804939>,
or mute the thread
